custom-vu/src/main/kotlin/jces1209/vu/MeasureType.kt

```diff
@@ -5,6 +5,9 @@ import com.atlassian.performance.tools.jiraactions.api.ActionType
 class MeasureType {
 
     companion object {
+        @JvmField
+        val CONTEXT_OPERATION_BOARD = ActionType("Context operation (Board)") { Unit }

```

Comment:
Adding it to the top, so it doesn't conflict with the bottom additions in other PRs, smart 🧠